tests/clients/test_dbclient.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sqlite3
+
+import pytest
+
+from location_extractor.clients import LocationDTO
+
+
+def test_populate_locations_table(dbclient):
+    if os.path.exists(dbclient.dbpath):
+        os.remove(dbclient.dbpath)
+
+    with pytest.raises(sqlite3.DatabaseError) as e:
```

Comment:
there is handy `matches` argument that you can pass to [`pytest.raises`](https://docs.pytest.org/en/latest/reference.html#pytest-raises) that matches text with `str(exc)`.
I don't know if it can be used in this case, but I just wanted to share this with you :D

```python
with pytest.raises(sqlite3.DatabaseError, matches='no such table: locations'):
    ...
```